nncf/experimental/common/tensor_statistics/collectors.py

```diff
@@ -412,23 +412,18 @@ def __init__(self, tensor_collectors: List[TensorCollector]) -> None:
 ##################################################
 
 
-class NoopReducer(TensorReducerBase):
+class RawReducer(TensorReducerBase):
     def __init__(self):
         super().__init__(inplace=False)
 
     def get_inplace_fn(self) -> Optional[InplaceInsertionFNType]:
         return None
 
-    def _reduce_out_of_place(self, x: List[TensorType]) -> List[TensorType]:
-        return x
-
-
-class RawReducer(NoopReducer):
-    def __init__(self):
-        super().__init__()
-
-    def __call__(self, x: List[Tensor]):
-        return self._reduce_out_of_place(x)
+    def _reduce_out_of_place(self, x: List[Tensor]) -> List[Tensor]:
+        output = []
+        for tensor in x:
+            output.append(tensor.clone())
```

Comment:
Could you please expand the test to check that tensors were actually cloned?

Reply:
done. test passes on PR and do not on develop